### ISSUE TITLE
Absent zona pellucida

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7394]
**Preferred term label:**
Absent zona pellucida

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lack of the oocyte extracellular matrix region (zona pellucida). This feature can be observed following follicular aspiration (retrieval of eggs for fertility treatments). 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:615774 
PMID:24670168  2/2

**Your nano-attribution (ORCID)**

**Comment by PaulNSchofield** [comment]
Im not sure this is really a phenotype as it probably reflects  physical manipulation.

**Comment by pnrobinson** [comment]
@PaulNSchofield I think the assumption is that the zona pellucida is not present even though it is normally visible after a procedure. Can you take a look at this?
https://pubmed.ncbi.nlm.nih.gov/24670168/

**Comment by pnrobinson** [comment]
@PaulNSchofield  -- I think this is the feature being referred to:
https://pubmed.ncbi.nlm.nih.gov/28886344/

During in vitro fertilization (IVF) treatment, oocyte retrieval is performed after ovarian stimulation via vaginal
puncture. Cumulus-oocyte complexes (COCs), which consist of cumulus cells surrounding the centrally located
oocyte, are isolated from the individual’s follicular fluid. ... However, of the six retrieved COCs, four were empty and two were found to contain degenerated oocytes lacking zonae pellucidae (ZP). 

If we add this description to the definition above, would it make sense?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000328
- label: Absent zona pellucida
- parents: [Abnormal zona pellucida morphology (HP:0020156)](https://hpo.jax.org/app/browse/term/HP:0020156)
- pmid: PMID:24670168
- def: Lack of the oocyte extracellular matrix region (zona pellucida). This feature can be observed following follicular aspiration (retrieval of eggs for fertility treatments).
- comment: During in vitro fertilization treatment, oocyte retrieval is performed after ovarian stimulation via vaginal puncture. Cumulus-oocyte complexes, which consist of cumulus cells surrounding the centrally located oocyte, are isolated from the individual's follicular fluid. This term refers to the finding of oocytes lacking zonae pellucidae.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7394
- synonyms:
